html/canvas/element/text-styles/META.yml

```diff
@@ -22,3 +22,13 @@ links:
           subtest: System fonts must be computed to explicit values
         - test: 2d.text.font.parse.complex.html
           subtest: 'Canvas test: 2d.text.font.parse.complex'
+    - product: chrome
+      url: https://crbug.com/1066953
+      results:
+        - test: 2d.text.draw.baseline.hanging.html
```

Comment:
Judging by https://wpt.live/html/canvas/element/text-styles/2d.text.draw.baseline.hanging.html and https://wpt.live/html/canvas/element/text-styles/2d.text.draw.baseline.ideographic.html I don't think these first two are about the same issue.

Reply:
Dropped:

```
        - test: 2d.text.draw.baseline.hanging.html
        - test: 2d.text.draw.baseline.ideographic.html
```